django/db/migrations/state.py

```diff
@@ -180,10 +179,10 @@ def __init__(self, real_apps, models, ignore_swappable=False):
                 # (only for the first issue). Error message should look like:
                 # "ValueError: Lookup failed for model referenced by
                 # field migrations.Book.author: migrations.Author"
-                msg = "Lookup failed for model referenced by field {field}: {model[0]}.{model[1]}"
-                raise ValueError(msg.format(field=operations[0][1], model=lookup_model))
+                msg = "Lookup failed for model: {model[0]}.{model[1]}"
+                raise ValueError(msg.format(model=lookup_model))
             else:
-                do_pending_lookups(model)
+                self.do_pending_operations(model)
```

Comment:
I'd certainly be tempted to get rid of this branch, unless someone can identify a case that hits it.